services/src/main/resources/db/changelog/lead-capture.db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <property name="autoIncrement" value="true" dbms="mysql,mssql,h2,sybase,db2,hsqldb"/>
+    <property name="autoIncrement" value="false" dbms="oracle,postgresql"/>
+
+    <!-- Definition of LEAD table -->
+    <changeSet author="exo-lead-capture" id="1.0.0-1">
+        <createTable tableName="ADDONS_LC_LEAD">
+
+            <column name="ID" type="BIGINT" autoIncrement="${autoIncrement}" startWith="1">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_LC_LEAD"/>
+            </column>
+            <column name="MAIL" type="NVARCHAR(150)">
```

Comment:
should be ate least 256 : https://tools.ietf.org/html/rfc5321#section-4.5.3

Reply:
Ok